R/pkg/R/install.R

```diff
@@ -201,14 +221,20 @@ directDownloadTar <- function(mirrorUrl, version, hadoopVersion, packageName, pa
   msg <- sprintf(fmt, version, ifelse(hadoopVersion == "without", "Free build", hadoopVersion),
                  packageRemotePath)
   message(msg)
-  downloadUrl(packageRemotePath, packageLocalPath, paste0("Fetch failed from ", mirrorUrl))
+  downloadUrl(packageRemotePath, packageLocalPath)
```

Comment:
It's still there. Both messages `Fetch failed` and error message from `download.file` were there before.

Before it is passing the url string several method calls deep, instead, I handle the error up in the stack and display the same message like in [here](https://github.com/apache/spark/pull/16589/files/9b21c1125998b09607dc40c57dcd579d322b6f6e#diff-48b5d60aba76122785461e1c2b125f51R121), and with url [here](https://github.com/apache/spark/pull/16589/files/9b21c1125998b09607dc40c57dcd579d322b6f6e#diff-48b5d60aba76122785461e1c2b125f51R176)